convert2rhel/unit_tests/actions/system_checks/duplicate_packages_test.py

```diff
@@ -61,6 +81,38 @@ def test_duplicate_packages_error(monkeypatch, output, expected, duplicate_packa
     )
 
 
+@pytest.mark.parametrize(
+    ("version_string", "output", "ret_code"),
```

Comment:
I think having some test case with unexpected values would be useful (like some different error)

Reply:
For this particular Warning only what's in the parametrize now will raise it - otherwise the check will expect that there were duplicate packages